src/senaite/core/browser/content/interpretationtemplates.py

```diff
@@ -0,0 +1,104 @@
+#
+# This file is part of SENAITE.CORE.
+#
+# SENAITE.CORE is free software: you can redistribute it and/or modify it under
+# the terms of the GNU General Public License as published by the Free Software
+# Foundation, version 2.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+# FOR A PARTICULAR PURPOSE. See the GNU General Public License for more
+# details.
+#
+# You should have received a copy of the GNU General Public License along with
+# this program; if not, write to the Free Software Foundation, Inc., 51
+# Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+#
+# Copyright 2018-2020 by it's authors.
+# Some rights reserved, see README and LICENSE.
+
+import collections
+
+from bika.lims import _
+from bika.lims.catalog import SETUP_CATALOG
+from bika.lims.utils import get_link_for
+
+from senaite.app.listing import ListingView
+
+
+class InterpretationTemplatesView(ListingView):
+    """Results Interpretation Templates listing view
+    """
+
+    def __init__(self, context, request):
+        super(InterpretationTemplatesView, self).__init__(context, request)
+
+        self.catalog = SETUP_CATALOG
+
+        self.contentFilter = {
+            "portal_type": "InterpretationTemplate",
+            "sort_on": "created",
+            "sort_order": "descending",
+        }
+
+        self.context_actions = {
+            _("Add"): {
+                "url": "++add++InterpretationTemplate",
+                "icon": "add.png"}
+            }
+
+        self.show_select_column = True
+
+        self.columns = collections.OrderedDict((
+            ("Title", {
+                "title": _("Title"),
+                "index": "sortable_title"}),
+            ("Description", {
+                "title": _("Description"),
+                "index": "Description"}),
+        ))
+
+        self.review_states = [
+            {
+                "id": "default",
+                "title": _("Active"),
+                "contentFilter": {"is_active": True},
+                "transitions": [],
+                "columns": self.columns.keys(),
+            }, {
+                "id": "inactive",
+                "title": _("Inactive"),
+                "contentFilter": {'is_active': False},
+                "transitions": [],
+                "columns": self.columns.keys(),
+            }, {
+                "id": "all",
+                "title": _("All"),
+                "contentFilter": {},
+                "columns": self.columns.keys(),
+            },
+        ]
+
+    def update(self):
+        """Update hook
+        """
+        super(InterpretationTemplatesView, self).update()
+
+    def before_render(self):
+        """Before template render hook
+        """
+        super(InterpretationTemplatesView, self).before_render()
+        # Don't allow any context actions
+        self.request.set("disable_border", 1)
```

Comment:
You can also make the content implement `IHideActionsMenu` (`from senaite.core.interfaces import IHideActionsMenu`)

Reply:
Good point! thanks